ipn/ipnlocal/serve.go

```diff
@@ -750,14 +750,22 @@ func (b *LocalBackend) serveWebHandler(w http.ResponseWriter, r *http.Request) {
 	http.Error(w, "empty handler", 500)
 }
 
+// stripMountPointFromError removes the root directory from the error message
+// to avoid exposing the local filesystem path.
+func stripMountPointFromError(err error, fileOrDir string) string {
+	errMsg := err.Error()
+	return strings.Replace(errMsg, fileOrDir, "./", 1)
```

Comment:
As a rule of thumb, whenever we deal with file paths, we should first look at https://pkg.go.dev/path/filepath to handle slashes correctly and since this is for windows, it should be `\` not `/` ?